src/components/Project/AragonClient.js

```diff
@@ -144,12 +147,10 @@ class AragonClient extends React.Component {
               <Entry>
                 <Module>
                   <div className="badge-container">
-                    <StyledBadge className="yellow">
-                      UX improvements
-                    </StyledBadge>
+                    <StyledBadge>New app</StyledBadge>
                     <AragonOneBadge>Aragon One</AragonOneBadge>
                   </div>
-                  <h3>Payroll app</h3>
+                  <h3>Aragon Payroll</h3>
```

Comment:
I notice that "Rewards app" above has the word "app" after it but with Aragon Agent and Aragon Payroll they do not, despite that these are also apps (and also that Aragon Agent and Aragon Payroll have the words "Aragon" before them but "Rewards" does not). If we're looking for consistency on this page then we should make sure we are referring to apps the same way.